README.md

```diff
@@ -1 +1,184 @@
+
+PostgresSQL
+Ruby on rails
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+    <li><a href="https://www.postgresql.org/">PSQL</a></li>
+  </ul>
+</details>
+<br>
+
+### Key Features <a name="key-features"></a>
+
+<details>
+  <ul>
+    <li>Add icon for each Category</li>
+    <li>Get total transaction for each category </li>
+    <li>Sign up using email and password</li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here you can visit my live demo :
+
+- [Click Here](https://mysite-7fzl.onrender.com/)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+<br>
+
+### Prerequisites
+
+<details>
+<p>In order to run this project you need:</p>
+  <ul>
+    <li>Ruby installed</li>
+    <li>rbenv installed</li>
+  </ul>
+</details>
+<br>
+
+### Setup
+
+Clone this repository to your desired folder:
+
+    git clone https://github.com/anas599/Budget-app.git
+
+cd Budget-App
+
+  <br>
+
+### Install
+
+<details>
+<p>Install this project with:</p>
+  <ul>
+    <li>  cd Budget-App</li>
+    <li>  run rails server</li>
+  </ul>
+</details>
+<br>
+
+### Usage
+
+<details>
+<p>To run the project, execute the following command:</p>
+  <ul>
+    <li>  cd Budget-App</li>
+    <li>  run rails server</li>
+  </ul>
+</details>
+<br>
+
+### Run tests
+
+<details>
+<p>To run tests, run the following command:</p>
+  <ul>
+    <li>rspec</li>
+  </ul>
+</details>
+<br>
+
+### Deployment
+
+<details>
+<p>You can deploy this project using:</p>
+  <ul>
+    <li>Heroku</li>
+    <li>Render</li>
+  </ul>
+</details>
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>👥 Authors <a name="authors"></a>
+
+👤 **ANAS ALSAMMARRAIE**
+
+- GitHub: [@anas599](https://github.com/anas599)
+- Twitter: [@anas599](https://twitter.com/anas599)
+- LinkedIn: [anas1993](https://linkedin.com/in/anas1993)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<details>
+  <ul>
+    <li>Add videos for new data</li>
+    <li>Add images of invoice</li>
+    <li>Update  style</li>
+  </ul>
+</details>
+<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/anas599/Budget-App.git/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like the project Please do not hesitate if you have any questions
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> I would like to thank Microverse for this project based learning
+
```

Comment:
- [ ] Based on the project requirement as shown below 👇🏾 , in the "Acknowledgements" section of the README, you should give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

#### Project requirement
![image](https://github.com/anas599/Budget-app/assets/84156714/89c2cfec-2936-40d5-aac1-4f85a1c3a2b0)